python/cugraph/cugraph/structure/convert_matrix.py

```diff
@@ -77,11 +78,19 @@ def from_edgelist(df, source='source', destination='destination',
                                     edge_attr, create_using, renumber)
 
     elif df_type is dask_cudf.core.DataFrame:
-        if create_using in [Graph, DiGraph]:
+        if create_using is None:
+            G = Graph()
+        elif isinstance(create_using, Graph):
+            attrs = {"directed": create_using.is_directed()}
+            G = type(create_using)(**attrs)
+        elif isinstance(create_using, DiGraph):
+            G = type(create_using)()
```

Comment:
While cugraph.DiGraph.is_directed would be fine because that check is inherited, the 'directed' keyword argument can't be passed into the DiGraph constructor, so `G=type(create_using)(**attrs)` would break